public/history.php

```diff
@@ -238,9 +236,11 @@ function convertDate() {
     }
     echo "</b><br>";
 
-    $memberSince = !empty($userSignedUp) ? getNiceDate(strtotime($userSignedUp), true) : '';
-    echo "Member since: " . $memberSince . "<br>";
-    echo "<br>";
+    // TODO replace this with created date
+//    $memberSince = !empty($userSignedUp) ? getNiceDate(strtotime($userSignedUp), true) : '';
+//    echo "Member since: " . $memberSince . "<br>";
+//    echo "<br>";
+
```

Comment:
open TODO

Reply:
`$userDetails->Created`. I almost made that change myself when working on this page yesterday.

Reply:
done in https://github.com/RetroAchievements/RAWeb/pull/1937/commits/8496628ac0e070b3e6ebf57dd3b6e35fa478dc7a